### ISSUE TITLE
Consider exposing intern/wolfram-fn in the public API

### ISSUE THREAD

**Issue by holyjak** [scicloj/wolframite#129]
While `wolframite.impl.wolfram-syms.intern/wolfram-fn` is primarily an internal function, it can be useful, f.ex. to create wrapper vars for symbols that are not in `wolframite.wolfram`, such as this case:

```clj
(def StringToMReal (wolframite.impl.wolfram-syms.intern/wolfram-fn 'Internal/StringToMReal))
(wl/eval (StringToMReal "12.34"))
;; => 12.34
```

It doesn't seem to belong to the core ns itself. Perhaps `wolframite.utils` or some such?!

**Comment by light-matters** [comment]
I don't think this should be in the 'api' as such because it seems more like an advanced developer option. Maybe this could be a subbrach of the API, e.g. `api.v1.advanced...`?

**Comment by holyjak** [comment]
I would wait with introducing new namespaces until we have more things to put there.

But since we now have the `api` ns, perhaps we could put the `intern/wolfram-fn` into the underlying `wolframite.core` ns?
